wallet/res/layout/custom_spinner_item.xml

```diff
@@ -1,11 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Deleting this file will cause merge problems when we pull `master` into `dashpay`.  DashPay currently has three Screens that still use this file.

Those screens will need to migrate away from the Spinner, but that may not happen before this story is merged into the dashpay branch.